src/client/src/containers/SelectOption/index.tsx

```diff
@@ -219,6 +240,24 @@ class SelectOption extends React.Component<Props, ISelectOptionState> {
     }
   }
 
+  public removePage(cardNumber: number) {
+    const {
+      options,
+      currentCardData,
+      cardTypeCount,
+      handleCountUpdate
+    } = this.props;
+    const { internalName } = options[cardNumber];
+    if (
```

Comment:
I know it's mostly for sanity, but is it necessary to perform these checks both here and in the `removeOption` function?

Reply:
Ideally it is not at all necessary. However, it just allows to make more independent functions that can be reused easily.